<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedactionDialog</class>
 <widget class="QDialog" name="RedactionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Redaction options</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="sizeGroup">
     <property name="title">
      <string>Symbols size</string>
     </property>
     <layout class="QFormLayout" name="sizeLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="aScaleLabel">
        <property name="text">
         <string>1:500</string>
        </property>
        <property name="buddy">
         <cstring>aScaleSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="aScaleSpin">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="bScaleLabel">
        <property name="text">
         <string>1:1000</string>
        </property>
        <property name="buddy">
         <cstring>bScaleSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="bScaleSpin">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cScaleLabel">
        <property name="text">
         <string>1:2000</string>
        </property>
        <property name="buddy">
         <cstring>cScaleSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="cScaleSpin">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="dScaleLabel">
        <property name="text">
         <string>1:5000</string>
        </property>
        <property name="buddy">
         <cstring>dScaleSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="dScaleSpin">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="toleranceGroup">
     <property name="title">
      <string>Tolerance</string>
     </property>
     <layout class="QFormLayout" name="toleranceLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="abstLabel">
        <property name="text">
         <string>Absolute tolerance</string>
        </property>
        <property name="buddy">
         <cstring>abstSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="abstSpin">
        <property name="specialValueText">
         <string>Disabled</string>
        </property>
        <property name="suffix">
         <string> mÂ²</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="prctLabel">
        <property name="text">
         <string>Percentage tolerance</string>
        </property>
        <property name="buddy">
         <cstring>prctSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="prctSpin">
        <property name="specialValueText">
         <string>Disabled</string>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="excludeGroup">
     <property name="title">
      <string>Excluded symbols</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="excludeEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="computeBox">
       <property name="text">
        <string>Compute symbols colisions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>aScaleSpin</tabstop>
  <tabstop>bScaleSpin</tabstop>
  <tabstop>cScaleSpin</tabstop>
  <tabstop>dScaleSpin</tabstop>
  <tabstop>abstSpin</tabstop>
  <tabstop>prctSpin</tabstop>
  <tabstop>excludeEdit</tabstop>
  <tabstop>computeBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RedactionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RedactionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
